great_expectations/datasource/fluent/spark_generic_partitioners.py

```diff
@@ -100,11 +102,13 @@ def batch_parameters_to_batch_spec_kwarg_identifiers(
         """Validates all the datetime parameters for this partitioner exist in `options`."""
         identifiers: Dict = {}
         for part in self.param_names:
-            if part not in options:
-                raise ValueError(f"'{part}' must be specified in the batch parameters")  # noqa: TRY003
-            identifiers[part] = options[part]
+            if part in options:
```

Comment:
Why can we remove the `ValueError` here?

Reply:
Partitioners know about sorting now, so users are free to not pass in any of the params, and just rely on sorting.

This made sense when we required users to pass in batch parameters to select the specific batch, but now that we let sorters do the heavy lifting if users don't pass in batch params, this was raising in cases when we didn't want it to.